src/test/suite/fixtures/recorded/languages/go/clearState.yml

```diff
@@ -0,0 +1,23 @@
+languageId: go
+command:
+  version: 0
+  spokenForm: clear state
+  action: clearAndSetSelection
+  targets:
+    - type: primitive
+      modifier: {type: containingScope, scopeType: statement, includeSiblings: false}
+initialState:
+  documentContents: "if true {\n\t\n}"
```

Comment:
It's a bit odd that you have these newlines.  Mind running your tests through the [autoformatter](https://github.com/pokey/cursorless-vscode/blob/main/docs/contributing/test-case-recorder.md#autoformatting)?

```sh
yarn run compile && node ./out/scripts/transformRecordedTests/index.js
```

Reply:
I formatted like that on purpose

Reply:
Shouldn't we just add the autoformatter as the part of the test recorder?

Reply:
The test recorder does output stuff formatted correctly, so I'm not sure how the test ended up this way

Reply:
@pokey I ran the automated formatter, but it didn't seem to have any affect :thinking:

Reply:
Hmm yeah I wonder if it's because the string includes tabs?  It's supposed to look more like this https://github.com/pokey/cursorless-vscode/blob/8d5e372ed494d996d382ffd450165d2c474e94e6/src/test/suite/fixtures/recorded/languages/typescript/clearFunk.yml#L10-L15

Reply:
I guess it's fine.  Probably not worth rewriting your tests.  Just maybe for the future try to use spaces